pkg/apis/admissionregistration/validation/validation.go

```diff
@@ -201,9 +201,16 @@ func validateWebhook(hook *admissionregistration.Webhook, fldPath *field.Path) f
 	}
 
 	cc := hook.ClientConfig
-	clientConfigErrors := webhook.ValidateWebhookClientConfig(fldPath.Child("clientConfig"),
-		cc.URL, false, cc.CABundle,
-		cc.Service != nil, cc.Service.Namespace, cc.Service.Name, cc.Service.Path)
+	var clientConfigErrors field.ErrorList
+	if cc.Service != nil {
+		clientConfigErrors = webhook.ValidateWebhookClientConfig(fldPath.Child("clientConfig"),
```

Comment:
two distinct call sites to this method, each of which only makes use of half the args, is a good sign that this should really be two different methods as described in the previous comment